### PR TITLE
Add GitKraken manifest

### PR DESCRIPTION
**Pull request by Bibz87**
## What type of Apps related PR is this? (check all applicable)

- [x] ✨ Add Manifest
- [ ] ♻️ Refactor
- [ ] 🐛 Bug Fix
- [ ] 👷 Optimization
- [ ] 🚩 Other

## Description
Manifest taken from scoop's _Extras_ bucket.

## Related Tickets & Documents
N/A

## [optional] What gif/emoji best describes this PR or how it makes you feel?
✨🔧

### Add/Modify a description in the manifest?

- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed

### Check URLs?

- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed

### Update to the latest version?

- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed

### Update Hashes?

- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed

### Format the json manifest?

- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed

### Notes in the manifest?

- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed

### Application Dependencies?

- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed

### Application Suggestions?

- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed

### Pre-installation steps to perform?

- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed

### Post-installation steps to perform?

- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed

### File persistence to perform?

- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed

### CODE REVIEW

**Review by alexchapar — APPROVED** [review]

**Comment by BotServiceAutomation** [comment]
All changes look good.

Wait for review from human collaborators.
### gitkraken

- [x] Description
- [x] License
- [x] Hashes
- [x] Checkver
- [x] Autoupdate
- [x] Autoupdate Hash Extraction

**Pull request merged**
